forem/forem
# Revert "Add (updated) mention autocomplete to post comments (#13061)"

## What type of PR is this? (check all applicable)

- [x] Feature Revert

## Description

The feature is working but it prevents navigation in the comment editor when using the up and down arrow keys. Further investigation is required to re-enable this feature.

![image](https://user-images.githubusercontent.com/833231/112354788-695a1000-8ca3-11eb-91f4-4158785604c3.png)

## Related Tickets & Documents

#13061

## QA Instructions, Screenshots, Recordings

Ensure that comments still work in posts (including nested comments) as well as comments in the notifications area.

You can see the QA notes in #13061 on how to generate notifications.

### UI accessibility concerns?

N/A

## Added tests?

- [x] No, and this is why: We're reverting to the old functionality which already (hopefully) had some tests.

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/), or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
      or in a [forem.dev](http://forem.dev) post
- [x] This change does not need to be communicated, and this is why not: We're reverting a feature that has not been communicated to the community yet.

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](gif_link)
